modules/common/adapters/adapter_gflags.cc

```diff
@@ -29,6 +29,8 @@ DEFINE_string(localization_topic, "/apollo/localization/pose",
               "localization topic name");
 DEFINE_string(planning_trajectory_topic, "/apollo/planning",
               "planning trajectory topic name");
+DEFINE_string(change_lane_topic, "/apollo/change_lane",
```

Comment:
Why do we need a separate topic for change lane?

Reply:
@davidhopper2003 We discussed internally and suggest making a topic for the remote control of ego vehicle. The change lane signal can be one of the signals that belongs to this topic.

Reply:
@davidhopper2003 @mfyanga Could you change the topic name from "/apollo/change_lane" to "/apollo/planning/pad" as we would like to use this topic for other manual inputs besides change lane for planning module?